prism/src/main/java/org/apache/falcon/jdbc/MonitoringJdbcStateStore.java

```diff
@@ -48,7 +48,7 @@ public void putMonitoredEntity(String entityName, String entityType) throws Falc
 
         MonitoredEntityBean monitoredEntityBean = new MonitoredEntityBean();
         monitoredEntityBean.setEntityName(entityName);
-        monitoredEntityBean.setEntityType(entityType);
+        monitoredEntityBean.setEntityType(entityType.toLowerCase());
```

Comment:
yes will make this change but we still need it in this class because the parameter entityType comes all the way through CLI and we have to convert it to lower case before sending it to DB in the where clause to get the results.